lib/openshift/project.rb

```diff
@@ -150,6 +150,11 @@ def get_pods(by:, **get_opts)
     end
     alias_method :pods, :get_pods
 
+    def get_jobs(by:, **get_opts)
+      Job.list(user: by, project: self, **get_opts)
+    end
+    alias_method :jobs, :get_jobs
```

Comment:
do we need this method? I'd rather remove all 3 of them but not sure if existing ones are used somewhere.

Reply:
These 3 methods are used in steps which have been used in feature files.